po/de.po

```diff
@@ -1529,9 +1528,8 @@ msgid "Sort current list (reverse)"
 msgstr "aktuelle Liste sortieren (umgekehrt)"
 
 #: src/keymap.cpp:406
-#, fuzzy
 msgid "Return to previous search results (if any)"
-msgstr "zum vorherigen Dialog zurückkehren/Beenden"
+msgstr "zum vorherigen Suchergebnis zurückkehren (sofern vorhanden)"
```

Comment:
Shouldn't this start with a capital letter?

Reply:
In fact, I first thought so as well when translating, but for some reason all these messages in the key bindings help screen are starting in lowercase in the German translations. Unless the first word is a noun, in which case German spelling rules requires nouns to start with an uppercase letter.